client/src/components/privateRoute.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Route, Redirect } from "react-router-dom";
+
+const PrivateRoute = ({component: Component, auth }) => (
+    <Route render={props => auth === true
+        ? <Component auth={auth} {...props} />
+        : <Redirect to={{pathname:'/home'}} />
```

Comment:
We don't have a login or register route (it's a modal) :( it goes to the home page, how will the user know they need to login/register?

Reply:
We could have it trigger a modal if we want! either that or we just hide the other navigation options other than the login modal so there's not a direct way to go to those private routes. Also would it make sense to differentiate the auth's in the component so that the one in the ternary references "isAuthenticated" so it's a little more explicit and reflects our state/context?

Reply:
lol! I forgot to take `auth` out. When I was developing global context state, the `auth` was used as double-check to confirm whether the context state worked after routing.

Reply:
In my opinion, instead of auth/login and auth/register, we can keep it directly /login and /register in routes.

Reply:
I replaced `auth` with `isAuthenticated`. Is that right?

Reply:
haha! I guess Eric was talking about deleting all the `auth`. Sorry, my bad.

Reply:
Looks good to me! As for how the private route could work, we could just have it as a kind of safety, and instead of a redirect when you click on one of the private routes, you could just hide them if "isAuthenticated" is false. Another option if that doesn't make sense would be perhaps to bring up the "open login/register modal" state to our global context and have that trigger if "isAuthenticated" is false.